provision/acc_provision/acc_provision.py

```diff
@@ -970,6 +997,56 @@ def generate_cert(username, cert_file, key_file):
             key_data = keyp.read()
     return key_data, cert_data, reused
 
+def generate_cko_cert(username, cert_file, key_file):
+    reused = False
+    if not exists(cert_file) or not exists(key_file):
+        info("Generating certs for cko")
```

Comment:
If the only reason we are creating a copy of this function is the above message, we can update the existing function with the logic to output the relevant message based on whether its for "kubernetes controller" or "cko". Also, rather than saying "cko", we can perhaps say "network-operator".

Reply:
Addressed